src/molecules/SubscriptionCompactAccordion/SubscriptionCompactAccordion.pcss

```
@@ -73,6 +74,10 @@
     padding: 1rem;
     margin: 0;
 
+    &--footer {
+      padding: 1.5rem 2rem 0rem !important;
+    }
+
     @media all and (min-width: 41em) {
       padding: 1.5rem 2rem;
     }
```

If you put it after the other padding, then the important-tag is not needed.
```suggestion
    @media all and (min-width: 41em) {
      padding: 1.5rem 2rem;
    }

    &--footer {
      padding: 1.5rem 2rem 0rem;
    }
```